components/brave_rewards/browser/publisher_info_database.cc

```diff
@@ -483,6 +483,10 @@ std::string PublisherInfoDatabase::BuildClauses(int start,
     ledger::PUBLISHER_EXCLUDE_FILTER::FILTER_ALL_EXCEPT_EXCLUDED)
     clauses += " AND pi.excluded != ?";
 
+  if (filter.percent == ledger::PUBLISHER_PERCENT_FILTER::FILTER_ZERO) {
+    clauses += " AND ai.percent != ?";
```

Comment:
see below - seems like <= would make more sense and just use pass an int (or double or whatever it is) defaulting to 100%